easybuild/easyblocks/p/paraver.py

```diff
@@ -88,8 +61,6 @@ def configure_step(self):
         wxwidgets = get_software_root('wxWidgets')
         if wxwidgets:
             wx_config = os.path.join(wxwidgets, 'bin', 'wx-config')
-        elif LooseVersion(self.version) >= LooseVersion('4.7'):
-            raise EasyBuildError("wxWidgets is not available as a dependency")
```

Comment:
This is still need, just not behind the `elif`?

Reply:
fixed in ee1ee1473